development/program.ttl

```diff
@@ -884,9 +987,25 @@ hpc:FunctionCall rdf:type owl:Class ;
                                    owl:onProperty hpc:calls ;
                                    owl:someValuesFrom hpc:Expression
                                  ] ,
+                                 [ rdf:type owl:Restriction ;
+                                   owl:onProperty hpc:destFunc ;
+                                   owl:someValuesFrom hpc:Function
+                                 ] ,
                                  [ rdf:type owl:Restriction ;
                                    owl:onProperty hpc:hasArgumentExpr ;
                                    owl:someValuesFrom hpc:Expression
+                                 ] ,
+                                 [ rdf:type owl:Restriction ;
+                                   owl:onProperty hpc:srcFunc ;
+                                   owl:someValuesFrom hpc:Function
+                                 ] ,
+                                 [ rdf:type owl:Restriction ;
+                                   owl:onProperty hpc:destFunc ;
+                                   owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                                 ] ,
+                                 [ rdf:type owl:Restriction ;
+                                   owl:onProperty hpc:srcFunc ;
+                                   owl:maxCardinality "1"^^xsd:nonNegativeInteger
                                  ] ;
                  rdfs:comment "A function call site, the same as the function call expression in AST" .
```

Comment:
This comment seems not accurate. Is this for a call graph, not for a call site? Please help fix it.

Reply:
This comment was in the ontology before it was modified by me: It looks like an hpc:Callsite is actually just an hpc:FunctionCall (specifically line 765, in which a callsite is marked as equivalent)